### DISCUSSION TITLE
评论消失，且无法更新评论

### DISCUSSION THREAD

**Discussion by misaka-fans** [walinejs/waline#1256] [Q&A] [answered]
没做更改，突然发现网页上以前的评论都消失了（管理端也没有），再发送也没反应，只有一个刷新按钮，刷新还是这样
![37CD90C2-BB5A-4F15-9AFC-FA838D01F45B](https://user-images.githubusercontent.com/104608787/183216260-5f953909-c5ce-466b-9068-d9bbb166a975.jpeg)

**Comment by misaka-fans** [comment, marked as answer]
解决了，发现是有人提交了一张图片（但好像出错了），然后全变成b64编码，占满一整页，内容太多，卡死了。不过请问这种会优化吗？:D

**Reply by lizheming** [reply]
https://github.com/walinejs/waline/pull/1257 增加了图片上传大小检测
